osidb/mixins.py

```diff
@@ -493,7 +493,7 @@ def acls_all(self):
         """
         get set of all ACLs
         """
-        return self.acls_read | self.acls_write | self.acls_internal
+        return self.acls_read | self.acls_write
```

Comment:
Why do you remove the internal ACLs from here? It clearly changes the method behavior in an unexpected way. All means all. If we need a method to get all without internal, we should rather write a new one.

Reply:
I am sorry for unclear commit message, all information is still here :)

`self.acls_read` contains `return self.acls_public_read | self.acls_embargo_read | self.acls_internal_read`, and `self.acls_write` contains `return(self.acls_public_write | self.acls_embargo_write | self.acls_internal_write)`.

Reply:
Oh I see! Then it makes sense to remove.